package.json

```diff
@@ -7,7 +7,7 @@
     "build": "npm run build:node && ng build --prod && ng run rpa-em-demo:server",
     "build:node": "cd api && npm run build",
     "start": "node ./dist/rpa-em-demo-api/server.js",
-    "start-dev-proxy": "ng serve --proxy-config proxy.config.json",
+    "start-dev-proxy": "ng serve --proxy-config proxy.config.json --host 0.0.0.0 ",
```

Comment:
what does that do?

Reply:
So, the --host 0.0.0.0 tells ng serve to bind to localhost. 
When its run in the context of a docker container, "localhost" is the ip address assigned by the docker network.

When you bind to port:3000 from the host it wasn't piping through. This allows it to whitelist your host basically